.ci/metrics/metrics.py

```diff
@@ -26,7 +26,67 @@ class JobMetrics:
     workflow_id: int
 
 
-def get_metrics(github_repo: github.Repository, workflows_to_track: dict[str, int]):
+@dataclass
+class GaugeMetric:
+    name: str
+    value: int
+    time_ns: int
+
+
+def get_sampled_workflow_metrics(github_repo: github.Repository):
+    """Gets global statistics about the Github workflow queue
```

Comment:
This currently will also pick up all the libc++ and Github hosted jobs. Is that intentional?

Reply:
I don't think so, since I'm filtering the lists depending on `WORKFLOWS_TO_TRACK`

Reply:
Ah, didn't see that. Sorry.